tests/unit/CmsZone.spec.ts

```diff
@@ -8,12 +8,66 @@ import CmsZone from '@/components/CmsZone.vue';
 import CmsPlugin from '@/plugins/cms';
 
 import { supressPromiseRejection } from './util';
+import { times } from 'lodash';
 
 Vue.compile = compileToFunctions;
 const localVue = createLocalVue();
 localVue.use(CmsPlugin);
+declare global {
```

Comment:
the mock intersection observer definitely feels like it belongs in a separate file